src/elements/copy.js

```diff
@@ -2,11 +2,13 @@ export default function (namespace) {
   const StringElement = namespace.getElementClass('string');
   const ArrayElement = namespace.getElementClass('array');
 
-  Object.defineProperty(ArrayElement.prototype, 'copy', {
-    get() {
-      return this.children.filter(item => item.element === 'copy');
-    },
-  });
+  if (!Object.getOwnPropertyNames(ArrayElement.prototype).includes('copy')) {
+    Object.defineProperty(ArrayElement.prototype, 'copy', {
```

Comment:
Do we actually need to monkey patch every single array element with a copy method? I would have though it only needs to be available on `category`, `resource` and `transition`? This could be moved so we don't have to monkey-patch external classes.